resip/stack/HEPSipMessageLoggingHandler.cxx

```diff
@@ -27,7 +27,11 @@ HEPSipMessageLoggingHandler::HEPSipMessageLoggingHandler(Data &captureHost, int
    mSocket = ::socket(PF_INET6, SOCK_DGRAM, IPPROTO_UDP);
 
    int no = 0;
-   setsockopt(mSocket, IPPROTO_IPV6, IPV6_V6ONLY, (void *)&no, sizeof(no));
```

Comment:
@sgodin - should we do it this way again, or something else?  I notice Dario removed this line and replaced it with the lines below